tests/units.ua

```diff
@@ -315,8 +315,7 @@ F ← ⨬(¯|+1|×10|∞)
 ⍤⤙≍ [[[1_2_3 5_6_7]] [[9_10_11 13_14_15]]] ◫¤2_3 +1°△4_4
 ⍤⤙≍ [
   [1_2_3 5_6_7]_[4_0_0 8_0_0]
-  [9_10_11 13_14_15]_[12_0_0 16_0_0]
-] ⬚0◫¤2_3 +1°△4_4
+  [9_10_11 13_14_15]_[12_0_0 16_0_0]] ⬚0◫¤2_3 +1°△4_4
```

Comment:
You seem to be using an old version of the formatter. Please remove this change.